### ISSUE TITLE
Starting a new neighborhood doesn't reset the main neighborhood stats to zero

### ISSUE THREAD

**Issue by manaswis** [ProjectSidewalk/SidewalkWebpage#313]
From PR https://github.com/ProjectSidewalk/SidewalkWebpage/pull/312:

<img width="1229" alt="jumped to deanwood" src="https://cloud.githubusercontent.com/assets/2873216/18300177/defb9a44-7494-11e6-9e36-73bb90073645.png">

<img width="1235" alt="new_mission" src="https://cloud.githubusercontent.com/assets/2873216/18323999/37b19f9e-7508-11e6-91da-bfae5e4c18bd.png">

Both 6.2 miles and 48 labels should have been zero

**Comment by kotarohara** [comment]
@manaswis Please describe what you think is the right behavior.

**Comment by manaswis** [comment]
The correct behavior should be: the stats should show zero for both labels and miles. In the above pic, the old neighborhood was Deanwood, and when the new neighborhood (Mount Pleasant) was loaded, the stats were from the old neighborhood.

**Comment by Soben713** [comment]
This is fixed in PR #337.

Just to clarify: Starting a new neighborhood does not necessarily mean the stats should be reset to zero. There are times when parts of the new neighborhood have been previously audited by the user (e.g., when a long street segment crosses the new neighborhood and is also part of another neighborhood that has been audited before). What I fixed was updating the status after a new neighborhood is started which may or may not be zero.

Here is a video of how it works now: https://www.youtube.com/watch?v=4YgHw-Z0-PQ&t=100s

We can close this issue now.

**Comment by jonfroehlich** [comment]
Maybe we should have some testing procedure before closing an issue. Like Manaswi can quickly test your fixes and you can quickly test her fixes--just as a sanity check.

Thanks for doing this!

**Comment by jonfroehlich** [comment]
Also, @Soben713, the video you uploaded isn't clear to me. What are you showing in the video?

**Comment by Soben713** [comment]
@jonfroehlich You are right. We actually decided to review each other's code before merging them to master. We can close the issues after the codes are reviewed and merged, but an alternative way (not so different, just what I'd rather to do) is to basically close the fixed issues when submitting a pull-request and re-open them if we find possible problems when reviewing the code. This just helps you feel a sense of accomplishment when you're done fixing a code. :)

And about the video, I just showed how it works right now. The breakpoint is on the line I added to the code and after it's executed the status is updated.
